src/6-0-intro.md

```diff
@@ -2,12 +2,51 @@
 
 Proper instrumentation is essential for monitoring, debugging, and optimizing your rollup in production. The Sovereign SDK provides comprehensive observability tools that help you understand your rollup's behavior and performance.
 
-This section covers:
-- **[Metrics](6-1-metrics.md)** - Track performance indicators and business metrics
-- **[Logging](6-2-logging.md)** - Debug and monitor your rollup's execution
+## Getting Started with Observability
 
-[TODO: Insert section on spinning up Grafana dashboards to monitor your rollup seamlessly]
+The [rollup starter](https://github.com/Sovereign-Labs/rollup-starter) repository includes a complete observability stack that gives you instant visibility into your rollup. With a single command, you can spin up a local monitoring environment:
```

Comment:
Couple things.

We need to make repo: https://github.com/Sovereign-Labs/sov-observability public.

Also, all dashboard and containers are defined there.

rollup starter repo has a command that pulls it and starts it